README.md

```diff
@@ -12,94 +12,68 @@ In this workshop you'll learn how to create a Redshift data warehouse that centr
+In this section we'll set up a new Redshift cluster
 
-- A Redshift cluster along with the security group rules necessary for accessing it from Etleap. 
-- An S3 bucket to host our data lake.
-- A Glue Catalog database to store data lake metadata.
-- An IAM role that will be assigned to the Redshift cluster and used to access the data lake.
-- An IAM user that will be used by Etleap to access the data lake.
-
-Log into your AWS account and [go to the page to create the stack for this workshop](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/quickcreate?templateUrl=https%3A%2F%2Fs3.amazonaws.com%2Fetleap-redshift-workshop%2Fcloudformation-templates%2Fcf-template.yaml&stackName=EtleapRedshiftDevDayStack). 
+Log into your AWS account and [go to the Redshift Console](https://console.aws.amazon.com/redshiftv2/home?region=us-east-1#clusters). 
 
 - Make sure the AWS region selected is N. Virginia (us-east-1).
-- Specify a root password for your Redshift cluster. This must consist of at least 8 alphanumeric characters only, and must contain a lower-case letter, an upper-case letter, and a number.
-- The other fields have sensible defaults, but feel free to modify as you see fit.
-
-After entering the all the parameter values: 
-- Click 'Next'.
-- On the next screen, enter any required tags, an IAM role, or any advanced options, and then click 'Next'.
-- Review the details on the final screen, Check the box for 'I acknowledge that AWS CloudFormation might create IAM resources' and then choose 'Create' to start building the resources.
-- Click 'Create stack' and wait for the creation to complete.
+- Click "Create Cluster" in top right-hand corner.
+- For the node type select `dc2.large`.
+- For the number of nodes, select `1`.
+- Under "Database configurations", enter a password that you'll remember. We will need this to connect to the cluster later on.
+- Leave all other settings as they are.
 
+Click "Create cluster".
+It will take 5-10 minutes for the cluster to start up.
 
-## 2. Connect Etleap to Redshift and the data sources
+## 2. Connect Etleap to Redshift and the data sources, and set up pipelines
 
-We'll ETL data from two different data sources into your Redshift data warehouse. One source, an S3 bucket, contains JSON-formatted click event data. The other, a MySQL database, contains information about users. 
+We'll ETL data from two different data sources into your Redshift data warehouse. One source, an SFTP source, contains JSON-formatted click event data. The other, a MySQL database, contains information about users. 
 
 The first thing you'll need to do is log into Etleap using the credentials that were provided to you via email. The email has the subject line 'Your Etleap Account'.
 
 In the rest of this section we'll connect Etleap to the data sources and Redshift destination, so that we can create pipelines in the next section.
 
+### 2.1. Set up the Redshift connection via the Partner Integration Console
```

Comment:
Would it make sense for this to be section 2 and the pipeline setup be section 3?

Reply:
I have made each source be its own section.